src/fcntl.rs

```diff
@@ -213,6 +213,18 @@ libc_bitflags!(
     }
 );
 
+/// Computes the raw fd consumed by a function of the form `*at`.
+#[cfg(any(
+    all(feature = "fs", not(target_os = "redox")),
+    all(feature = "process", any(target_os = "android", target_os = "linux"))
+))]
+pub(crate) fn at_rawfd(fd: Option<RawFd>) -> raw::c_int {
+    match fd {
+        None => libc::AT_FDCWD,
+        Some(fd) => fd,
```

Comment:
FYI, this pattern can be more succinctly represented as `fd.unwrap_or(libc::AT_FDCWD)`.

Reply:
Done